site/content/blog/2022-12-01-whats-new-in-svelte-december-2022.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "What's new in Svelte: December 2022"
+description: "Rounding the corner to SvelteKit 1.0"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+SvelteKit 1.0 is just around the corner! With [99% of the milestone issues completed](https://github.com/sveltejs/kit/milestone/2), there's a lot of new changes from the last month to cover...
+
+Let's get to it!
+
+## What's new in SvelteKit
+- Use the `willUnload` property to find out if the navigation will result the app being unloaded (full page reload/closing/leaving to another page). ([#6813](https://github.com/sveltejs/kit/pull/6813))
+- `__data.json` requests now use the `Vary` header, which allows one cached response for each variation of the `x-sveltekit-invalidated` header, ensuring we cache matching responses for all invalidation scenarios ([#7532](https://github.com/sveltejs/kit/pull/7532))
```

Comment:
We realized that not all platforms/CDNs support this header, so this will change - I'll reword it to something more generic

```suggestion
- `__data.json` requests now allows for caching while ensuring we cache matching responses for all invalidation scenarios ([#7532](https://github.com/sveltejs/kit/pull/7532))
```